contracts/Core/storage/RewardStorage.sol

```diff
@@ -5,6 +5,4 @@ import "../../lib/Structs.sol";
 
 contract RewardStorage {
     uint256 public blockReward;
```

Comment:
Right now for storing only single param, is it worth to have separate storage contract, Can we move it to RewardManager directly as global variable ?

Reply:
This doesn't impact anything but
As its inheritance, at last, it remains same

Reply:
I don't remember why we are keeping it in RewardManager, 
Its ideal place is in parameters.

Reply:
@abhishekvispute could you please create an issue for this?

Reply:
> I don't remember why we are keeping it in RewardManager,
> Its ideal place would be in parameters.

yes I noticed as well. it needs to be in parameters